### DISCUSSION TITLE
Create a logo for helmfile

### DISCUSSION THREAD

**Discussion by yxxhero** [helmfile/helmfile#38] [Ideas]
Create a logo for helmfile

**Comment by mumoshu** [comment]
I saw several people posted logos at our slack channel (`#helmfile` in https://slack.sweetops.com/)
Probably we can use one of those with credits and thanks!

**Reply by yxxhero** [reply]
https://files.slack.com/files-pri/TB2A18NRE-F03BFCNC001/asset_2_2x.png  this is good.

**Comment by pathob** [comment]
@yxxhero @mumoshu I guess the ideas are always pretty much the same. I would like to offer this one:

![Artboard 1](https://user-images.githubusercontent.com/4580157/181858119-d204b277-62d0-42d7-8e11-a7c85d1cdcf8.png)

**Comment by yxxhero** [comment]
@pathob Thanks very much. Can you create a PR for this. put this in logo dir.

**Comment by pathob** [comment]
@yxxhero The PR is there but let's wait for an opinion of @mumoshu
